src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -114,7 +114,7 @@ export const VegaWallet = () => {
           )}
         </WalletCardHeader>
         <WalletCardContent>{child}</WalletCardContent>
-        <WalletCardContent>{version}</WalletCardContent>
+        {!version ? <WalletCardContent>{version}</WalletCardContent> : null}
```

Comment:
This seems weird. If we have no version we display the version?

Reply:
fixed